### ISSUE TITLE
slurm daemons started too early

### ISSUE THREAD

**Issue by sjpb** [stackhpc/ansible-role-openhpc#17]
In `runtime.yml` the slurm services are started before munge and the OHPC packages are installed. Starting the daemons potentially will run any jobs stored in state files, so this is the wrong way round. It works ok on a fresh install as there won't be jobs stored, but trips up e.g. if autoscaling uses this playbook to add nodes.

More broadly there may also be other requirements not part of this role (e.g. distributed filesystems) which should be satisfied before starting/restarting the services.
